var/spack/repos/builtin/packages/redis/package.py

```diff
@@ -18,39 +18,48 @@ class Redis(MakefilePackage):
     """
 
     homepage = "https://redis.io"
-    urls = [
-        "https://download.redis.io/releases/redis-7.0.5.tar.gz",
-        "https://github.com/redis/redis/archive/refs/tags/7.0.5.tar.gz",
-    ]
+    url = "https://github.com/redis/redis/archive/refs/tags/7.4.0.tar.gz"
```

Comment:
We should keep the `urls` list in place, as the previous versions are not on github, or at least with different hashes. I don't quite see from the history why the GitHub archive url was added, but we usually prefer archives on project websites (if available) over GitHub archives since the archives may not contain all required components (submodules, generated autotools files).

Reply:
When keeping the `urls` there is also no need to change the version number in the url. That's automatically deduced and replaced with the requested version number.